### ISSUE TITLE
Support for Python 3

### ISSUE THREAD

**Issue by ExtensiveJS** [morpheus65535/bazarr#283]
Would it be possible to support v3.x of Python? Maintaining multiple versions gets a bit tricky and support for the latest (major atleast) version would be great.

**Comment by morpheus65535** [comment]
I’ve made an attempt to port Bazarr to Python 3 last week bit there’s a lot of change to be done and full test to run afterward. We don’t have the man power to do this for now but I’ll keep it as a low priority enhancement. Thanks!

**Comment by ngosang** [comment]
https://pythonclock.org/

**Comment by morpheus65535** [comment]
Let's talk about it again in 11 months... ;-)

**Comment by ngosang** [comment]
https://www.phoronix.com/scan.php?page=news_item&px=Ubuntu-Debian-Python-2-Process

**Comment by dutch2005** [comment]
https://medium.com/@boxed/moving-a-large-and-old-codebase-to-python3-33a5a13f8c99

TL;DR --> convert it via SIX to Py2/py3 compatible then "remove" the py2 stuff

plus python2 is EOL 1-1-2020...
